examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -48,7 +47,7 @@
    "outputs": [],
    "source": [
     "# set env variable for data\n",
-    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/Users/sspeichert/Documents/LyftData/\"\n",
```

Comment:
this is your local path :)

Reply:
Thanks for checking so throughly and sorry that it was necessary! all mentions fixed now!